sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_models.py

```diff
@@ -0,0 +1,609 @@
+        self._thumbprint = thumbprint
+        self._tags = kwargs.get("tags", None)
+
+    @classmethod
+    def _from_certificate_item(cls, certificate_item):
+        # type: (models.CertificateItem) -> Certificate
+        """Construct a Certificate from an autorest-generated CertificateItem"""
+        return cls(
+            attributes=certificate_item.attributes,
+            cert_id=certificate_item.id,
+            thumbprint=certificate_item.x509_thumbprint,
+            tags=certificate_item.tags,
+        )
+
+    @property
+    def id(self):
+        # type: () -> str
+        pass
+
+    @property
+    def name(self):
+        # type: () -> str
+        return self._vault_id.name
+
+    @property
+    def thumbprint(self):
+        # type: () -> bytes
+        return self._thumbprint
+
+    @property
+    def enabled(self):
+        # type: () -> bool
+        return self._attributes.enabled
+
+    @property
+    def not_before(self):
+        # type: () -> datetime
+        return self._attributes.not_before
+
+    @property
+    def expires(self):
+        # type: () -> datetime
+        return self._attributes.expires
+
+    @property
+    def created(self):
+        # type: () -> datetime
+        return self._attributes.created
+
+    @property
+    def updated(self):
+        # type: () -> datetime
+        return self._attributes.updated
+
+    @property
+    def recovery_level(self):
+        # type: () -> str
+        return self._attributes.recovery_level
+
+    @property
+    def vault_url(self):
+        # type: () -> str
+        return self._vault_id.vault_url
+
+    @property
+    def tags(self):
+        # type: () -> Dict[str, str]
+        return self._tags
+
+
+class Certificate(CertificateBase):
+    def __init__(self, attributes, cert_id, thumbprint, kid, sid, policy, cer, **kwargs):
+        # type: (models.CertificateAttributes, str, bytes, str, str, models.CertificatePolicy, bytes, Mapping[str, Any]) -> None
+        super(Certificate, self).__init__(attributes, cert_id, thumbprint, **kwargs)
+        self._kid = kid
+        self._sid = sid
+        self._policy = policy
+        self._cer = cer
+
+    @classmethod
+    def _from_certificate_bundle(cls, certificate_bundle):
+        # type: (models.CertificateBundle) -> Certificate
+        """Construct a Certificate from an autorest-generated CertificateBundle"""
+        return cls(
+            attributes=certificate_bundle.attributes,
+            cert_id=certificate_bundle.id,
+            thumbprint=certificate_bundle.x509_thumbprint,
+            kid=certificate_bundle.kid,
+            sid=certificate_bundle.sid,
+            policy=CertificatePolicy._from_certificate_policy_bundle(certificate_bundle.policy),
+            cer=certificate_bundle.cer,
+            tags=certificate_bundle.tags,
+        )
+
+    @property
+    def kid(self):
+        # type: () -> str
+        return self._kid
+
+    @property
+    def sid(self):
```

Comment:
Should `kid` and `sid` have friendlier names?

Reply:
For now, I think these are the names that got decided across all languages. We can discuss this further in the Arch board meeting.